content/en/users/specific/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Additional User Documentation
+linkTitle: Cannot find it?
```

Comment:
It's good to see things being sanitised/simplified, but I've mixed feelings about the "Cannot find it?" link, not sure how meaningful/helpful it would be for users.
But I don't have a proper/better proposition for now

Reply:
others, @enolfc, what do you think of this? Any suggestion? Should we just merge like this?

Reply:
Enol suggested `Other docs` or maybe `Other documentation`.
What do you think?

Reply:
This looks like a dark corner of the docs that we cannot organize or fit anywhere else :) Which is the truth, really. The title "Other..." is neutral, a sort of acknowledgment that we gave up on the stuff there. The problem I have with it is that it's not catchy enough. It will be easily overlooked. I would like a sidebar that can be navigated without having a prior knowledge of the docs structure. In summary, I am not against it, but I do not like it either.

Reply:
should we add comms team for advice?

Reply:
"Other..." does not fit in our docs structure. We have services organized in sections (Compute, Data, etc.). And we have 2 pseudo sections: Getting Started, Tutorials. The title "Cannot find it?" at least was masking that dark corner as a pseudo section.

Reply:
Looking at the community specific page, that is just regular business for al VOs/projects. That page should not even exist. Maybe replace it with a tutorial that applies to all VOs/projects. And the provider specific page should be just a paragraph at the end of the Open Stack page. Let me know if you think this is better.

Reply:
The idea was that we could collect provider-specific documentation for all services, being them HTC or Cloud. The initial idea was not to only have a link to a given doc, but potentially some details that may be relevant.
That said I'm fine with adding the link to providers-pages somewhere near in the cloud compute page. And in case at some point we have HTC-specific doc we could had them there.

I agree that the page for disaster mitigation is quite generic, the only specific info seems to be the VO name to use and provider/site details. That said like for the OpenStack network ID or project ID it's information than can be fairly useful for a given community when you look at helping them getting access/using the resources. But not sure where it's the best to have this/document this.
I guess it's too be discussed with the CST people, ie. @glarocca, @sebastian-luna-valero (that created this page) and the CST team.

Reply:
Asked Ilaria for an opinion. @glarocca and @sebastian-luna-valero what do you think?

Reply:
> I agree that the page for disaster mitigation is quite generic, the only specific info seems to be the VO name to use and provider/site details. That said like for the OpenStack network ID or project ID it's information than can be fairly useful for a given community when you look at helping them getting access/using the resources. But not sure where it's the best to have this/document this. 

In the _Getting Started_ section, I guess. There it should be described the way we offer access, a guide on how somebody can gain access, including this information. Makes no sense to duplicate it with different VO names. And it definitely cannot be thrown in the root of the docs tree.

Reply:
Discussed with @thebe14 and I agree to move it in the Getting Started section. Thx.

Reply:
I checked with Giuseppe and moved the communities section under Getting Started. The other I set as draft, will incorporate it into the cloud compute section update I am perparing. I suggest we merge like this for now.

Reply:
Nota bene, the OpenStack and CLI nodes will disappear from under Getting Started too in the future, going back under Cloud Compute.

Reply:
OK, fine with me, will merge once @enolfc and @glarocca will have approved the PR.